prime-dotnet-webapi/Migrations/20210329212828_HALikeCareSettingWithJobTitle.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class HALikeCareSettingWithJobTitle : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "HealthAuthorityCode",
+                table: "OboSite",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "JobTitle",
+                table: "OboSite",
+                nullable: false,
+                defaultValue: "");
+
+            migrationBuilder.CreateIndex(
+                name: "IX_OboSite_HealthAuthorityCode",
+                table: "OboSite",
+                column: "HealthAuthorityCode");
+
+            migrationBuilder.AddForeignKey(
+                name: "FK_OboSite_HealthAuthorityLookup_HealthAuthorityCode",
+                table: "OboSite",
+                column: "HealthAuthorityCode",
+                principalTable: "HealthAuthorityLookup",
+                principalColumn: "Code",
+                onDelete: ReferentialAction.Restrict);
+
+            // Update the Job Titles of all Obo Sites of an Enrollee
```

Comment:
Migrations look good to me, tested.

Reply:
Actually, I dont think you should drop the Job table here, or if you do then you also have to remove Job from everywhere in the backend code. It fails to load enrollees after running this migration.

Reply:
Thanks @AnaisHebert for checking.  I'll remove the Job table deletion.  Maybe removing Job "artifacts" can be a clean-up ticket next sprint?

Reply:
Clean up of artifacts can definitely occur next sprint.

Reply:
Addressed